src/prepack-node.js

```diff
@@ -34,14 +34,21 @@ export function prepackStdin(options: PrepackOptions = defaultOptions, callback:
   process.stdin.on("data", function(code) {
     fs.readFile(sourceMapFilename, "utf8", function(mapErr, sourceMap) {
       if (mapErr) {
-        console.warn(`No sourcemap found at ${sourceMapFilename}.`);
+        //if no sourcemap was provided we silently ignore
+        if (sourceMapFilename !== "") console.warn(`No sourcemap found at ${sourceMapFilename}.`);
         sourceMap = "";
       }
       let filename = "no-filename-specified";
       let serialized;
       try {
         serialized = prepackSources(
-          [{ filePath: filename, fileContents: code, sourceMapContents: sourceMap }],
+          [
+            {
+              filePath: filename,
+              fileContents: code,
+              sourceMapContents: sourceMap,
+            },
+          ],
```

Comment:
This reformatting seems unnecessary.

Reply:
Prettier added it. I will manually correct it.